Fix double dots and asciidoc formatting in tables (#4765)
strimzi/strimzi-kafka-operator: documentation/modules/appendix_crds.adoc

|string array
|maxConnectionCreationRate     1.2+<.<a|The maximum connection creation rate we allow in this listener at any time. New connections will be throttled if the limit is reached.Supported only on Kafka 2.7.0 and newer.
|integer
|maxConnections                1.2+<.<a|The maximum number of connections we allow for this listener in the broker at any time. New connections are blocked if the limit is reached.
|integer
|preferredNodePortAddressType  1.2+<.<a|Defines which address type should be used as the node address. Available types are: `ExternalDNS`, `ExternalIP`, `InternalDNS`, `InternalIP` and `Hostname`. By default, the addresses will be used in the following order (the first one found will be used):
* `ExternalDNS`
* `ExternalIP`
* `InternalDNS`
* `InternalIP`
* `Hostname`

> This field can be used to select the address type which will be used as the preferred type and checked first. In case no address will be found for this address type, the other types will be used in the default order.This field can be used only with `nodeport` type listener.

Reviewer: ```suggestion
This field is used to select the preferred address type, which is checked first. If no address is found for this address type, the other types are checked in the default order. This field can only be used with `nodeport` type listener.
```
